src/test/java/no/unit/alma/commons/HttpStatusExceptionTest.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.alma.commons;
+
+import no.unit.alma.generated.error.WebServiceResult;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+class HttpStatusExceptionTest {
+
+
+    @Test
+    void testHttpStatusException() {
+        int status = 23;
+        String statusText = "sample status text";
+        String method = "sample method";
+        String url = "https://www.example.com";
+        String responseBody = "sample body";
+        String resultAsString = "sample result string";
+        WebServiceResult webServiceResult  = new WebServiceResult();
+        webServiceResult.setResult(resultAsString);
+
+        HttpStatusException httpStatusException = new HttpStatusException(status, statusText, method, url, responseBody, webServiceResult);
+
+        assertEquals(status, httpStatusException.getStatus());
+        assertEquals(statusText, httpStatusException.getStatusText());
+        assertEquals(method, httpStatusException.getMethod());
+        assertEquals(url, httpStatusException.getUrl());
+        assertEquals(responseBody, httpStatusException.getResponseBody());
+        assertEquals(webServiceResult.getResult(), httpStatusException.getWebServiceResult().getResult());
+        assertTrue(httpStatusException.isWebserviceResult());
+    }
+
```

Comment:
Come on: å teste det også er vel en easy catch:


```
    public boolean isWebserviceResult() {
        return webServiceResult != null;
    }
```

Reply:
hmm. den ble kanskje forenklet litt mye på sist checkstyle- refraktorering

Reply:
får jo 100% på denne, men har laget en ekstra-test for deg

Reply:
Stemmer det. Feil Klasse, det gjelder ResponseFilter.